aten/src/ATen/native/native_functions.yaml

```diff
@@ -3301,6 +3301,10 @@
   dispatch:
     CompositeImplicitAutograd: rms_norm_symint
 
+- func: _rms_norm_fused(Tensor input, int normalized_shape_ndim, Tensor weight, float eps) -> Tensor
```

Comment:
I am not sure if there is some naming convention, or if it's just coincidence, but when I search the word "fused" in native_functions.yaml, it is always before the name of the function. If there is such naming convention, then this should be changed to `_fused_rms_norm`